tests/func/test_remote.py

```diff
@@ -170,6 +170,41 @@ def test(self):
         self.assertEqual(ret, 0)
 
 
+def test_with_checksum_jobs(dvc):
+    ret = main(["remote", "add", "with_checksum_jobs", "s3://bucket/name"])
+    assert ret == 0
+    ret = main(
+        ["remote", "modify", "with_checksum_jobs", "checksum_jobs", "100"]
+    )
+    assert ret == 0
+    ret = main(["config", "core.checksum_jobs", "200"])
+    assert ret == 0
+
+    dvc.config.load()
+    remote = Remote(dvc, name="with_checksum_jobs")
+    assert remote.checksum_jobs == 100
+
+
+def test_without_checksum_jobs(dvc):
+    ret = main(["remote", "add", "without_checksum_jobs", "s3://bucket/name"])
+    assert ret == 0
+    ret = main(["config", "core.checksum_jobs", "200"])
+    assert ret == 0
+
+    dvc.config.load()
+    remote = Remote(dvc, name="without_checksum_jobs")
+    assert remote.checksum_jobs == 200
+
+
+def test_with_checksum_default(dvc):
```

Comment:
We could do these tests as unit ones in tests/unit/remote/test_base.py, which should be much simpler and lightweight.

Reply:
Just to clarify, this is totally mergable, but could be improved. So your call.

Reply:
I sounds as a call :) so I will rework it